test/domains/sungeun/rect/at_operator.bad

```diff
@@ -1,4 +1,4 @@
-at_operator.chpl:6: syntax error: Invalid token: near '@'
+at_operator.chpl:6: syntax syntax error: Invalid token: near '@'
```

Comment:
😨  "syntax syntax"?

Reply:
Oof -- why are all my `.bad` mistakes not being caught by paratest? I thought we check that the output of the respective `.chpl` still matches the `.bad`?

Reply:
I don't think futures are tested by a "regular" paratest. Not sure how to enable it, though.